docs/extras.rst

```diff
@@ -16,6 +16,8 @@ and even harder to regularly test. Hypothesis is always tested against the lates
 compatible version and each package will note the expected compatibility range. If
 you run into a bug with any of these please specify the dependency version.
 
+There are seperate pages for :doc:`django` and :doc:`numpy`.
```

Comment:
Probably this should include pandas in this sentence too? (Though it's the same page for numpy).

In general I would like us to move to smaller pull requests (I know, hypocrisy, but I'm trying to be better too), so this should probably be in its own thing, but I'll allow it. :-)

Reply:
These resolve to the page title, so "Hypothesis for Django Users" and "Hypothesis for the Scientific Stack".

Reply:
Ah, right! In that case that's fine.